transactions/README.md

```diff
@@ -0,0 +1,52 @@
+# Transactions
+
+## Transaction object
+
+```
+{   
+    typeOfCurrency: string,
+    typeOfTransaction: string,
+    userID: string,
+    amount: number,
+    dateTime: unix-timestamp
+}
+```
+
+## **Requests**
+
+|               Route                |           Description           |
+| :--------------------------------: | :-----------------------------: |
+|      [GET /fetchLatest/:username?noOfRecords](#fetch-latestTransactions)      | Returns noOfRecords latest Transactions in the system  for username provided|
+----
+## **GET /latestTransactions**
+
```

Comment:
The endpoint URL in table (line 19) and line 21 don't match

Also, can it be short like `transactions/latest/:username`?

Reply:
revised URL be like : /transactions/fetch/kratika 
                                /transactions/fetch/kratika?n=5&o=DESC